build.gradle

```diff
@@ -55,6 +56,8 @@ buildscript {
         classpath "me.champeau.gradle:jmh-gradle-plugin:$jmh_plugin_version"
         classpath "org.jetbrains.kotlinx:kotlinx.benchmark.gradle:$benchmarks_version"
         classpath "org.jetbrains.kotlinx:binary-compatibility-validator:$validator_version"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath "com.android.tools.build:gradle:3.4.2"
```

Comment:
consider extracting the version to `gradle.properties`. btw It looks like this version is outdated

Reply:
Taking into account the compatibility table (see here https://developer.android.com/studio/releases/gradle-plugin) we can't use newer versions.

Reply:
Done.